package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "grunt-cache-bust",
   "description": "Bust static assets from the cache using content hashing",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
By [semver](http://semver.org/) rules it's a new feature that doesn't break backwards compatibility, so should be `1.1.0` - dw, I'll change it when it's merged, just an FYI